data/settings/icon-grid-C.json.in

```diff
@@ -3,24 +3,41 @@
     "chromium-browser.desktop",
     "org.gnome.Nautilus.desktop",
     "libreoffice-writer.desktop",
-    "rhythmbox.desktop",
-    "com.endlessm.photos.desktop",
+    "libreoffice-calc.desktop",
+    "libreoffice-impress.desktop",
     "com.endlessm.encyclopedia.en.desktop",
-    "net.minetest.Minetest.desktop",
+    "eos-link-facebook.desktop",
+    "eos-link-whatsapp.desktop",
     "eos-link-youtube.desktop",
     "eos-folder-media.directory",
-    "eos-folder-games.directory",
-    "eos-folder-work.directory",
     "eos-folder-curiosity.directory",
-    "eos-folder-social.directory"
+    "eos-folder-games.directory",
+    "eos-folder-social.directory",
+    "net.minetest.Minetest.desktop",
+    "com.endlessm.cooking.en.desktop",
+    "rhythmbox.desktop"
   ],
   "eos-folder-media.directory" : [
+    "com.endlessm.photos.desktop",
     "shotwell.desktop",
     "org.gnome.Totem.desktop",
-    "net.sourceforge.Audacity.desktop",
     "org.tuxpaint.Tuxpaint.desktop",
+    "net.sourceforge.Audacity.desktop",
+    "com.endlessm.world_literature.en",
     "org.gimp.Gimp.desktop"
   ],
+  "eos-folder-curiosity.directory" : [
+    "com.endlessm.celebrities.en.desktop",
+    "com.endlessm.myths.en.desktop",
+    "com.endlessm.math.en.desktop",
+    "com.endlessm.howto.en.desktop",
+    "com.endlessm.travel.en.desktop",
+    "com.endlessm.dinosaurs.en.desktop",
+    "com.endlessm.health.en.desktop",
+    "kde4-org.kde.Marble.desktop",
```

Comment:
I'll remove for now - this is actually from me having the KDE repos installed on my system - but you're right, we really should enable maps.